django/contrib/admin/checks.py

```diff
@@ -104,7 +104,7 @@ def _check_raw_id_fields_item(self, obj, model, field_name, label):
             return refer_to_missing_field(field=field_name, option=label,
                                           model=model, obj=obj, id='admin.E002')
         else:
-            if not isinstance(field, (models.ForeignKey, models.ManyToManyField)):
+            if not isinstance(field, models.ForeignKey) and not field.many_to_many:
                 return must_be('a ForeignKey or ManyToManyField',
```

Comment:
I'm hesitant to remove "ManyToManyField" in the message since that's the common case. I'm not sure if adding "or a field with many_to_many=True" to all the messages is worth the additional verbosity.